extensions/ql-vscode/src/databases/config/db-config-store.ts

```diff
@@ -32,6 +32,8 @@ import {
 } from "../db-item";
 
 export class DbConfigStore extends DisposableObject {
+  static readonly databaseConfigFileName = "workspace-databases.json";
```

Comment:
(Optional) I personally like to be explicit with the visibility of properties

```public static readonly databaseConfigFileName = "workspace-databases.json";`

Reply:
👍